swe_cp_WB.m

```diff
@@ -513,52 +512,44 @@ function swe_cp_WB(SwE)
       else
          descrip = sprintf('unrestricted fitted data (%04d)', i);
       end
-      VYWB(i) = swe_create_vol(sprintf('YfittedWB_%04d.img',i), DIM, M, descrip);
+      VYWB(i) = swe_create_vol(sprintf('YfittedWB_%04d%s',i,file_ext), DIM, M, descrip);
```

Comment:
Missed this one too in my table of revised names; let's go for `swe_vox_fit_y{y#}`